packages/connector-chrome/src/chrome-launcher.ts

```diff
@@ -110,30 +104,12 @@ export class CDPLauncher extends Launcher {
         await writeFileAsync(this.pidFile, JSON.stringify({ pid: browserInfo.pid, port: browserInfo.port || this.port }, null, 4));
     }
 
-    public async launch(url: string): Promise<BrowserInfo> {
-        const cdpLock = 'cdp.lock';
+    public async launch(url: string, callback: Function): Promise<BrowserInfo> {
```

Comment:
did you forget to remove the `callback`? I don't see you are using it anywhere. Also, having a callback and returning a Promise is weird.